frontend/packages/dev-console/locales/zh/devconsole.json

```diff
@@ -0,0 +1,314 @@
+  "Max Pods": "最大 Pod",
+  "The upper limit for the number of pods that can be set by autoscaler.": "此自动扩展可设置的 pod 数量上限。",
+  "Concurrency Target": "并行目标",
+  "Defines how many concurrent requests are wanted per instance of the application at a given time (soft limit) and is the recommended configuration for autoscaling. If not specified, will be defaulted to the value set in the cluster config.": "定义在给定时间（软限制）每个应用程序实例需要多少个并发请求，它是自动扩展的推荐配置。如果没有指定，将默认使用集群配置中设置的值。",
+  "Concurrency Limit": "并行限制",
+  "Limits the amount of concurrent requests allowed into one instance of the application at a given time (hard limit), and is configured in the revision template. If not specified, will be defaulted to the value set in the cluster config.": "限制在定时间（硬件限制）允许并发请求到应用程序的一个实例中的并发请求量，它在修订模板中配置。如果没有指定，将默认使用集群配置中设置的值。",
+  "Warning: the application grouping already exists.": "警告：应用程序组已存在。",
+  "A unique name given to the application grouping to label your resources.": "为应用程序组指定一个唯一名称以标记您的资源。",
+  "Select an application for your grouping or {{UNASSIGNED_LABEL}} to not use an application grouping.": "选择用于分组的应用程序或{{UNASSIGNED_LABEL}}以不使用应用程序分组。",
+  "Application Name": "应用程序名称",
+  "General": "常规设置",
+  "Project Name": "项目名称",
+  "A unique name for the project.": "项目的唯一名称。",
+  "Project Display Name": "项目显示名称",
+  "Project Description": "项目描述",
+  "A unique name given to the component that will be used to name associated resources.": "分配给组件的唯一名称，用于命名相关资源。",
+  "Builder Image": "构建器镜像",
+  "Detecting recommended builder images...": "检测推荐的构建器镜像...",
+  "Builder image(s) detected.": "检测到的构建器镜像。",
+  "Recommended builder images are represented by": "推荐的构建器镜像由表示",
+  "icon": "图标",
+  "Unable to detect the builder image.": "无法检测构建器镜像。",
+  "Select the most appropriate one from the list to continue.": "从列表中选择最合适的一个来继续。",
+  "Builder Image Version": "构建器镜像版本",
+  "Builder": "构建器",
+  "Sample repository:": "样本仓库：",
+  "Create": "创建",
+  "Deploy Image": "部署镜像",
+  "Hide Advanced Git Options": "隐藏高级 Git 选项",
+  "Show Advanced Git Options": "显示高级 Git 选项",
+  "Git Reference": "Git 参考",
+  "Optional branch, tag, or commit.": "可选 branch、tag 或 commit。",
+  "Context Dir": "Context Dir",
+  "Optional subdirectory for the application source code, used as a context directory for build.": "可选的应用程序源代码的子目录，用作构建的上下文目录。",
+  "Dockerfile": "Docker",
+  "Dockerfile Path": "Dockerfile 路径",
+  "Allows the builds to use a different path to locate your Dockerfile, relative to the Context Dir field.": "允许构建使用不同的路径来定位您的 Dockerfile，相对于 Context Dir 字段。",
+  "Container Port": "容器端口",
+  "Port number the container exposes.": "容器公开的端口号。",
+  "Validating": "验证",
+  "Validated": "已验证",
+  "URL is valid but cannot be reached. If this is a private repository, enter a source secret in Advanced Git Options": "URL 有效但无法访问。如果这是一个私有仓库，在高级 Git 选项中输入源 secret",
+  "Git": "Git",
+  "Git Repo URL": "Git Repo URL",
```

Comment:
"Git Repo URL" needs to be translated into Chinese.

Reply:
This was not addressed.

Reply:
I can confirm it's not changed - getting clarification from @rh-tfu whether it should be left as English or translated

Reply:
we can leave it in English